README.md

```diff
@@ -7,7 +7,103 @@
 [chat-badge]: https://img.shields.io/discord/1082332781146800168.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2&style=flat-square
 [chat-url]: https://discord.gg/kKV8ZxyzY4
 
+# 🏁🏁 Build Status 🏁🏁
+You can view the current build status for the different packages [here](https://prefix-dev.github.io/reproducible-builds/).
 
+## Introduction
+This project aims to see if we can create reproducible builds for a variety of software packages.
+The project is based on definitions of the [Reproducible Builds](https://reproducible-builds.org/) project.
```

Comment:
wdyt about adding a tl;dr about what is for us a reproducible build?

Reply:
Good one what do you suggest?

Reply:
Something along the lines:

tl;dr
We validate that `rattler-build` is a reproducible build tool by ensuring that it can recreate identical packages given the same recipe and build environment. We verify this by comparing the `SHA256` hash of the original build and the rebuild. If the hashes differ, we inspect the packages with `diffoscope` to identify the differences. Common issues affecting reproducibility include unsorted files, differing archive timestamps, and reset script permissions during the archiving step.

Reply:
Ok I've added that and a bit extra!